### PR TITLE
Remove double underscores

### PR DESCRIPTION
**Pull request by eabovre**
Corrected unnecessary double underscores

### CODE REVIEW

File: pds_pipelines/models/upc_models.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def create_table(keytype, *args, **kwargs):
 
 
 class DataFiles(Base):
-    _tablename_ = 'datafiles'
+    ___tablename___ = 'datafiles'
```
Comment (AustinSanders) [inline pds_pipelines/models/upc_models.py]: Now these are triple underscored :(

Comment (eabovre) [reply pds_pipelines/models/upc_models.py]: Oh my word, why did I make this so hard! I just fixed it, please let me know if there is anything else I breezed over!

File: pds_pipelines/models/upc_models.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ class Keywords(Base):
 
 class Meta(object):
     # Enforce compound primary key constraint
-    __table_args__ = (PrimaryKeyConstraint('upcid', 'typeid'),)
+    _table_args_ = (PrimaryKeyConstraint('upcid', 'typeid'),)
```
Comment (AustinSanders) [inline pds_pipelines/models/upc_models.py]: One last change -- table args still requires double underscores.

Comment (eabovre) [reply pds_pipelines/models/upc_models.py]: I am actually laughing out loud about how many times I've said "all right, this has to be the last one"

**Pull request merged**